src/blocks/HeaderBlock/mobile.tsx

```diff
@@ -52,28 +55,26 @@ export const CrossBurger = styled('div')(
 )
 
 const MiddleBurger = styled('div')(({ isOpen, isClosing }: TogglableState) => ({
-  position: 'absolute',
   width: '100%',
   top: '50%',
   left: 0,
   right: 0,
   height: 3,
   backgroundColor: isOpen && !isClosing ? 'transparent' : 'currentColor',
   transition: 'background-color 300ms',
-  transform: 'translateY(-1.5px)',
+  transform: 'translateY(-1px)',
```

Comment:
when i come to think of it i think this should be half of `height` which is 3 and that's why it was 1.5. any particular reason for changing it?

Reply:
This is the half-pixel thing that made it look off on different screens. I guess we could try to cover it by using half-pixels on the other places as well? But this looks correct I think 🤷‍♀